client/app/dedicatedCloud/datacenter/host/dedicatedCloud-datacenter-host.routes.js

```diff
@@ -1,6 +1,6 @@
 angular
   .module('App')
-  .config(($stateProvider) => {
+  .config(/* @ngInject */ ($stateProvider) => {
```

Comment:
You could've added them to `controllers` too :)

Reply:
It's actually been done for most controllers !

https://github.com/ovh-ux/ovh-manager-dedicated/pull/1022/commits/25b20b765a609931e23f28d2be5b771c85c8eaa2